src/website/src/pages/ec/components/forms/select/docs/usage.mdx

```diff
@@ -36,27 +38,27 @@ import { Paragraph, Anatomy, Link } from '@ecl/website-components';
 | heading          | yes       | heading describes the options in the Select Menu                                                   |
 | helper text      | no        | helper text information assists a user in using Select Menu                                        |
 | dropdown list    | yes       | dropdown in the collapsed state with placeholder giving instructions for using the Dropdown Select |
-| action indicator | yes       | **displays listed items on click**                                                                 |
+| action indicator | yes       | displays listed items on click                                                                     |
 
 ### Conditional (missing input)
 
-| Elements           | Mandatory | Description                                                             |
-| ------------------ | --------- | ----------------------------------------------------------------------- |
-| optional indicator | yes       | indicator stating **if** input fields are **optional**                  |
-| error message      | yes       | a message informing the user **if** there is a problem with their entry |
+| Elements           | Mandatory | Description                                                         |
+| ------------------ | --------- | ------------------------------------------------------------------- |
+| optional indicator | yes       | indicator stating if input fields are optional                      |
+| error message      | yes       | a message informing the user if there is a problem with their entry |
 
 ## Do's
 
-- write short, distinct and indicative headings for the **items listed inside the dropdown**
-- **use dropdown select menu labels as click targets (clicking the label will trigger the action)**
-- indicate whether the input is **optional**
-- **make use of helper text** if there are further directions or hints the users may need in completing their goal (**example**: _you must be a current resident of this country_)
-- **write specific and clear error message texts, so users understand how to address the error**
+- write short, distinct and indicative headings for the items listed inside the dropdown
+- use dropdown select menu labels as click targets (clicking the label will trigger the action)
+- indicate whether the input is optional
+- make use of helper text if there are further directions or hints the users may need in completing their goal (example: _you must be a current resident of this country_)
+- write specific and clear error message texts, so users understand how to address the error
 
 ## Don'ts
 
-- **don't use as navigation**
-- **don't use it for multiple select**
+- don't use as navigation
+- don't use it for multiple select - use Multi-dropdown select instead
```

Comment:
The multi-dropdown select is a link in the wiki, I think it could be achieved in the mdx as well with the Link component 
```
<Link to="/ec/components/forms/search-form/usage/" standalone>Europa search form</Link>
```
I found this example in the repo

Reply:
Reading on later, maybe a simple link with an anchor, but info is so close that it could unnecessary.